### DISCUSSION TITLE
CVE-2024-54143是否影响本固件

### DISCUSSION THREAD

**Discussion by quner2026** [immortalwrt/immortalwrt#1626] [General]
OpenWrt项目团队于12月6日发布安全公告，披露了其固件升级存在严重安全漏洞(CVE-2024-54143)。该漏洞由日本Flatt Security公司安全研究员RyotaK发现并报告。

漏洞源于两个关键问题的组合：系统在构建固件时未对用户提供的软件包名称进行适当过滤，导致可能被注入恶意命令；同时，请求哈希机制仅使用SHA-256哈希值的前12个字符，大大降低了安全性，使攻击者可能通过制造哈希碰撞来污染合法固件。

这一漏洞可能影响所有使用OpenWrt在线固件升级服务的用户。攻击者无需认证即可利用该漏洞，通过注入命令和制造哈希碰撞，使合法的构建请求收到预先生成的恶意固件以完成入侵。

OpenWrt团队在漏洞报告后迅速采取行动，已于12月4日完成修复并部署。检查显示过去七天内未发现被利用痕迹。建议用户及时更新到最新版本以确保系统安全。

请问ImmortalWRT的部分功能会受此影响吗？

**Comment by 1715173329** [comment]
https://t.me/ctcgfw_project_openwrt/35
https://t.me/nanopi_r2s/745

**Reply by quner2026** [reply]
万分感谢！
